### DISCUSSION TITLE
Миграция с других АТС

### DISCUSSION THREAD

**Discussion by plastilincheg** [mikopbx/Core#55] [Q&A] [answered]
Планируется ли создать или может есть инструмент миграции, например с FreePBX, Elastix, Issabel? С максимальным сохранением функционала, что был развернут на сервере планируемом для перехода.

**Comment by boffart** [comment, marked as answer]
Пока есть только 
[Конвертация истории звонков FreePBX в MikoPBX](https://wiki.mikopbx.com/far:conv_cdr_freepbx_to_mikopbx)
[Объединение MIKOPBX и FreePBX (PJSIP)](https://wiki.mikopbx.com/faq:mikopbx_freepbx)
[Объединение MIKOPBX и FreePBX (IAX2)](https://wiki.mikopbx.com/faq:mikopbx_freepbx_iax2)
